<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditLaunchersDialog</class>
 <widget class="QDialog" name="EditLaunchersDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>750</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Edit Launchers</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>680</y>
     <width>741</width>
     <height>35</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="launchersNote">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>520</y>
     <width>771</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Tips: &lt;/p&gt;&lt;ul style=&quot;margin-top: 0px; margin-bottom: 0px; margin-left: 0px; margin-right: 0px; -qt-list-indent: 1;&quot;&gt;&lt;li style=&quot; margin-top:12px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;Use drag-and-drop in the launcher list to re-order the items. &lt;/li&gt;&lt;li style=&quot; margin-top:0px; margin-bottom:12px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;To add a standard program launcher, simply right-click a running program on the dock panel then tick &amp;quot;Pinned&amp;quot;. &lt;/li&gt;&lt;/ul&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="textFormat">
    <enum>Qt::RichText</enum>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="remove">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove Selected</string>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="removeAll">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>340</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Remove All</string>
   </property>
   <property name="icon">
    <iconset theme="edit-delete">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="addSeparator">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>180</y>
     <width>361</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Separator</string>
   </property>
   <property name="icon">
    <iconset theme="list-add">
     <normaloff>.</normaloff>.</iconset>
   </property>
  </widget>
  <widget class="QComboBox" name="addSystemCommand">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>100</y>
     <width>361</width>
     <height>44</height>
    </rect>
   </property>
   <property name="frame">
    <bool>true</bool>
   </property>
   <item>
    <property name="text">
     <string>Add System Command</string>
    </property>
   </item>
  </widget>
  <widget class="QListWidget" name="launchers">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>350</width>
     <height>490</height>
    </rect>
   </property>
   <property name="dragDropMode">
    <enum>QAbstractItemView::InternalMove</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>EditLaunchersDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>EditLaunchersDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
